labels-owners.yaml

```diff
@@ -44,8 +49,11 @@ labels:
       - krishnadurai
       - kunmingg
       - lluunn
+      - yanniszark
```

Comment:
How come your adding yourself as an owner of centraldashboard? Are you actually working on centraldashboard?

Reply:
I haven't, I think you're not viewing the diff correctly.
This is the area/istio label.